content/docs/v2/_dev/operations/monitoring.md

```diff
@@ -50,3 +50,139 @@ Logs by default go to `stderr` in plain text format. For production deployment l
+- **Network Usage**: Monitor network I/O for data ingestion and storage operations
+
+#### Disk I/O (for components with local storage)
+- **Disk Usage**: Monitor disk space utilization
+- **Disk I/O**: Track read/write operations per second and latency
+
+### Go Runtime Metrics
+
+Jaeger v2 components are built on OpenTelemetry Collector, which exposes standard Go runtime metrics:
+
+#### Garbage Collection
+- **GC Duration**: Monitor garbage collection pause times
+- **GC Frequency**: Track how often garbage collection occurs
+
+#### Goroutines
+- **Goroutine Count**: Monitor the number of active goroutines
+- **Goroutine Growth**: Watch for unexpected increases that may indicate leaks
+
+#### Memory Statistics
+- **Heap Usage**: Monitor Go heap memory usage
+- **Memory Allocations**: Track allocation patterns
+
+### OpenTelemetry Collector Metrics
+
+Jaeger v2 is built on OpenTelemetry Collector. The key metrics to monitor are:
+
+#### Span Processing
+- **Span Ingestion**: Monitor spans received
+  - `otelcol_receiver_accepted_spans`: Total spans accepted by receivers
+  - `otelcol_receiver_refused_spans`: Spans refused (indicates data loss)
+- **Span Export**: Monitor spans sent to storage
+  - `otelcol_exporter_sent_spans`: Successfully exported spans
+  - `otelcol_exporter_send_failed_spans`: Failed export attempts
+
+#### Component Health
+- **Receiver Health**: Monitor data ingestion components by receiver type
+- **Exporter Health**: Monitor storage export components by storage backend
+
+### Query Performance
+
+Monitor Jaeger query service performance:
+
+#### Response Times
+- **Query Latency**: Monitor UI and API query response times
+- **Query Volume**: Track query patterns and frequency
+
+#### Storage Backend
+- Monitor your specific storage backend (Elasticsearch, Cassandra, etc.) according to its documentation
+
+### Prometheus Alerts
+
+For Jaeger v1 deployments, the Jaeger repository includes a comprehensive set of Prometheus alerting rules in the [monitoring mixin](https://github.com/jaegertracing/jaeger/tree/main/monitoring/jaeger-mixin). These production-tested alerts cover:
+
+- **Service availability**: Alerts for when Jaeger components are down
+- **Performance degradation**: Alerts for high latency and error rates
+- **Capacity issues**: Alerts for resource exhaustion and queue saturation
+- **Data loss**: Alerts for dropped spans and storage failures
+
+The monitoring mixin provides:
+- **Alert rules** (`alerts.libsonnet`) - Ready-to-use Prometheus alerting rules
+- **Grafana dashboards** - Pre-built dashboards for visualization
+- **Runbooks** - Documentation for responding to alerts
+
+For detailed setup instructions, see the [monitoring mixin documentation](https://github.com/jaegertracing/jaeger/tree/main/monitoring/jaeger-mixin).
+
+### Custom Alerting Considerations
+
+For Jaeger v2 deployments or custom alerting needs, consider setting up alerts for:
+
+#### Critical Issues
```

Comment:
these sections seem to be repeating the section above. There is no difference between monitoring and alerting as far as I am concerned, they go together.